src/test/java/com/amazon/opendistroforelasticsearch/commons/InjectSecurityTest.java

```diff
@@ -77,7 +79,7 @@ public void testInjectUser() {
         assertEquals("plugin", threadContext.getTransient("ctx.name"));
 
         try (InjectSecurity helper = new InjectSecurity("test-name", settings, threadContext)) {
-            helper.inject("joe", "ops-role,logs-role");
+            helper.inject("joe", Arrays.asList("ops-role,logs-role"));
```

Comment:
This is considered as 1 element string. Do you want 2 elements in the array?

Reply:
good catch, fixed.